features/steps/payment_steps.py

```diff
@@ -0,0 +1,80 @@
+from behave import given, when, then
+from flask_api import status
+from app.db.interface import PaymentService
+import json
+
+@given(u'the following payments')
```

Comment:
huh, you know, i didn't even think about it. lifted the first example from class demo which used them and never looked back. but glancing back at behave docs now and theres definitely no requirement to use unicode.